py2hwsw/scripts/user_api/api.py

```diff
@@ -331,7 +332,11 @@ def create_interface_from_text(interface_text):
 
     Attributes:
         interface_text (str): Short notation text. Object attributes are specified using the following format:
-            type [-p prefix] [-m mult] [-w width]
+            genre [-d direction] [-p prefix] [-m mult] [-f file_prefix] [-pm portmap_port_prefix] [-w WIDTH_W:val]+ -[-P PARAM:val]+
```

Comment:
In the future, I'm not sure if we will support 'genre'. We may add the interface's subclasses directly to the API instead.